data-node/src/main/java/org/graylog/datanode/docs/printers/ConfigFileDocsPrinter.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+package org.graylog.datanode.docs.printers;
+
+import org.apache.commons.lang.WordUtils;
+import org.graylog.datanode.docs.ConfigurationEntry;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.util.Arrays;
+import java.util.Locale;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class ConfigFileDocsPrinter implements DocsPrinter {
+
+    public static final String DATANODE_CONFIG_HEADER = """
```

Comment:
There is only a tiny difference between graylog.conf and datanode.conf header. Maybe it would be enough to provide string headline and use the whole header as a template?

Reply:
Sounds good.